parent/pom.xml

```diff
@@ -40,6 +40,7 @@
     <version.awaitility>4.2.2</version.awaitility>
     <version.bouncycastle>1.78.1</version.bouncycastle>
     <version.camunda>7.22.0-alpha4</version.camunda>
+    <version.camunda-license-check>2.9.0</version.camunda-license-check>
```

Comment:
❓ I assume that this is the recent version that has been published/released now on a open maven repo?

Reply:
Yes! It was moved with https://github.com/camunda/camunda-license-check/pull/34